src/shared/components/VideoPreviewBase/VideoPreviewBase.styles.tsx

```diff
@@ -220,3 +275,35 @@ export const CoverPlaceholder = styled(Placeholder)`
   width: 100%;
   height: 100%;
 `
+
+export const CoverCheckboxContainer = styled.div`
+  position: absolute;
+  top: ${sizes(2)};
+  left: ${sizes(2)};
+  padding: ${sizes(2)} ${sizes(2)};
+`
+
+export const KebabMenuIconContainer = styled.div`
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: 40px;
+  height: 40px;
+  cursor: pointer;
+  position: relative;
+  border-radius: 100%;
+  transition: all ${transitions.timings.regular} ${transitions.easing};
+
+  path:not([fill='none']) {
```

Comment:
That's an unusual selector, what's happening here?

Reply:
This is to target the path's tags inside the svg's so they transition nicely and to change the color of the icon on hover

Reply:
but actually just realised I dont need the `:not([fill='none']) ` part the first one